.buildkite/pipelines/quality-gates/pipeline.kibana-tests.yaml

```diff
@@ -15,7 +15,7 @@ env:
   ENVIRONMENT: ${ENVIRONMENT?}
 
 steps:
-  - label: ":pipeline::grey_question::seedling: Trigger Kibana Tests for ${ENVIRONMENT}"
+  - label: ":pipeline::grey_question::seedling: Trigger Kibana Tests"
```

Comment:
Good catch! I think I removed this because I had the wrong idea in my head - I thought this pipeline will trigger all other environments with that one command. Now I know this pipeline will be called for every env, and this file will just re-route the request to a different pipeline.

Reply:
re-added